index.html

```diff
@@ -55,11 +58,12 @@ <h2 class="black heading">Hey there.I'm <br> Precious <br>
             </ul>
            <a class="angle-down" href=""><img src="icons/Disabled.png" alt=""></a>
        </section>
```

Comment:
- [ ] Kindly, make this `angle-down` to be in the center of the page

![02222222](https://user-images.githubusercontent.com/79058364/161378705-9512eb1a-4c7a-45e5-b6b4-df255d07d0e6.png)

Reply:
For this section, It is the social media icons that need centering and not the `angle-down`